### PR TITLE
Update/theme

### PR DESCRIPTION
**Pull request by TerminalFi**
Proposed updated landing page

image

image

### CODE REVIEW

**Comment by FichteFoll** [comment]
Will take a look at this some other time, but some initial feedback/questions:

1. We should still link to the official site somewhere, either the landing page or the docs.
2. Not sure yet how I feel about the toggle always having the same color. I understand that you can see its "active" state by the look of the page but I imagine "dark &lt;toggle be here&gt; light" to function better.
   The orange is also very attention-grabbing, moreso than it should, probably.

Edit: moved relevant topics into review below.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                                <div class="feature-inner">
+                                    <a href="//lsp.sublimetext.io" class="feature-icon">
+                                        <img class="asset-light" src="dist/images/sublime-lsp.png" alt="Sublime LSP">
+                                        <img class="asset-dark" src="dist/images/sublime-lsp.png" alt="Sublime LSP">
+                                    </a>
+                                    <div class="feature-content">
+                                        <a href="//lsp.sublimetext.io">
+                                            <h3 class="feature-title mt-0">Sublime LSP Documentation</h3>
+                                        </a>
+                                        <p class="text-sm mb-0">Sublime Text's community driven implementation for LSP
```
Comment (FichteFoll) [inline index.html:75]:
```suggestion
                                        <p class="text-sm mb-0">Sublime Text's community-driven implementation for LSP
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                                            alt="Sublime Text Documentation">
+                                    </a>
+                                    <div class="feature-content">
+                                        <a href="//docs.sublimetext.io">
+                                            <h3 class="feature-title mt-0">Unofficial Sublime Text Documentation</h3>
+                                        </a>
+                                        <p class="text-sm mb-0">The Sublime Text Community Documentation is a community
+                                            effort at documenting the <a href="https://sublimetext.com/">Sublime
+                                                Text</a> text editor and accompanying its <a
+                                                href="https://sublimetext.com/docs/3/">official documentation</a>.</p>
```
Comment (FichteFoll) [inline index.html]: These are some weird line breaks to be honest, but it's not like it will be changed much or similar.

File: docs/index.html [hunk 3]
```diff
@@ -1,47 +1,109 @@
-<!DOCTYPE HTML>
-<html lang="en">
+<!DOCTYPE html>
```
Comment (FichteFoll) [inline docs/index.html]: How about using a proper github pages workflow via a second branch and the accompanying github action? That setup has been tested and is also what we use in the docs repo.

It saves us from 
1. having the same files (assets) twice in the repo,
2. forgetting to update the built files,
3. ignoring half of the changes on pull requests.

File: docs/index.html [hunk 4]
```diff
@@ -1,47 +1,109 @@
...
+                                            alt="Sublime Text Documentation">
+                                    </a>
+                                    <div class="feature-content">
+                                        <a href="//docs.sublimetext.io">
+                                            <h3 class="feature-title mt-0">Unofficial Sublime Text Documentation</h3>
+                                        </a>
+                                        <p class="text-sm mb-0">The Sublime Text Community Documentation is a community
+                                            effort at documenting the <a href="https://sublimetext.com/">Sublime
+                                                Text</a> text editor and accompanying its <a
+                                                href="https://sublimetext.com/docs/3/">official documentation</a>.</p>
```
Comment (FichteFoll) [inline docs/index.html]: The official docs are linked here so you can ignore my earlier comment reagrding that.

File: docs/index.html [hunk 5]
```diff
@@ -1,47 +1,109 @@
...
+    <div class="body-wrap boxed-container">
+        <header class="site-header">
+            <div class="container">
+                <div class="site-header-inner">
+                    <div class="brand header-brand">
+                        <h1 class="m-0">
+                            <div class="lights-toggle">
+                                <input id="lights-toggle" type="checkbox" name="lights-toggle" class="switch">
+                                <label for="lights-toggle" class="text-xs"><span>Turn me <span
+                                            class="label-text">dark</span></span></label>
```
Comment (FichteFoll) [inline docs/index.html]: Moving this from my separate comment:

> Not sure yet how I feel about the toggle always having the same color. I understand that you can see its "active" state by the look of the page but I imagine "dark <toggle be here> light" to function better.
> The orange is also very attention-grabbing, moreso than it should, probably.

File: docs/index.html (hunk 3 above)
Comment (TerminalFi) [reply docs/index.html]: Will work on this

File: docs/index.html (hunk 5 above)
Comment (TerminalFi) [reply docs/index.html]: Will work on this

**Pull request merged**